cmd/object-api-options.go

```diff
@@ -66,7 +66,7 @@ func getDefaultOpts(header http.Header, copySource bool, metadata map[string]str
 	if crypto.S3.IsRequested(header) || (metadata != nil && crypto.S3.IsEncrypted(metadata)) {
 		opts.ServerSideEncryption = encrypt.NewSSE()
 	}
-	opts.ProxyRequest = header.Get(xhttp.MinIOSourceProxyRequest) == "true"
+	opts.ProxyRequest = header.Get(xhttp.MinIOSourceProxyRequest)
```

Comment:
```suggestion
	opts.ProxyRequest = header.Get(xhttp.MinIOSourceProxyRequest)
```

why change this? @poornas

Reply:
reverted back to using bool